templates/Pages/Settings.CodeBehind/Views/SettingsPagePage.xaml.cs

```diff
@@ -8,7 +8,7 @@ namespace Param_ItemNamespace.Views
 {
     public sealed partial class SettingsPagePage : Page, System.ComponentModel.INotifyPropertyChanged
     {
-        //// TODO WTS: Add other settings as necessary. For help see https://github.com/Microsoft/WindowsTemplateStudio/blob/master/docs/pages/settings.md
+        //// TODO WTS: Add other settings as necessary. For help see https://github.com/Microsoft/WindowsTemplateStudio/blob/master/docs/pages/settings-codebehind.md
```

Comment:
mvvm and mvvmlight don't have this todo?

Reply:
@mrlacey, ping :)

Reply:
Yes, the link in the codebehind version of the template now points to the code behind specific version of the docs. This [exists in the dev branch](https://github.com/Microsoft/WindowsTemplateStudio/blob/dev/docs/pages/settings-codebehind.md) but hasn't yet made it to master. It will when this change goes public too.

I didn't change the MVVM Basic & MVVM Light versions and they still point to the [generic page](https://github.com/Microsoft/WindowsTemplateStudio/blob/dev/docs/pages/settings.md) (with links to framework specific versions) as the alternative would be to add two new composition templates and that felt like a lot more effort than was worth including for the sake of saving someone clicking a link.

Reply:
MVVM Basic & MVVM LIght have the TODO in the ViewModel as that's where the majority of changes are needed.